src/pages/MonthlyGood/KivaAsExpert.vue

```diff
@@ -114,13 +114,13 @@ export default {
 			return `${this.yearsSinceStartCalc} years, ${this.amountFunded ? `${this.amountFunded} billion in impact` : 'billions of dollars in impact'}`;
 		},
 		description() {
-			let desc = 'With billions of dollars in loans funded';
+			let desc = 'With billions of dollars';
 			if (this.amountFunded) {
 				desc = `With nearly ${this.amountFunded} billion`;
```

Comment:
Could we have billions here?

Reply:
`billion` should be correct in the cases I can think of, it's also what we showed before.